src/main/java/edu/kis/powp/jobs2d/drivers/adapter/Jobs2dDriverToDrawerAdapter.java

```diff
@@ -0,0 +1,41 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+/**
+ * driver adapter to drawer with several bugs.
+ */
+public class Jobs2dDriverToDrawerAdapter implements Job2dDriver {
+    private int startX = 0, startY = 0;
+
+	private DrawPanelController drawPanelController;
+
+    public Jobs2dDriverToDrawerAdapter(DrawPanelController drawPanelController) {
+        super();
+        this.drawPanelController = drawPanelController;
+    }
+
+    @Override
+    public void setPosition(int x, int y) {
+        this.startX = x;
+        this.startY = y;
+    }
+
+    @Override
+    public void operateTo(int x, int y) {
+        ILine line = LineFactory.getBasicLine();
```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with LineFactory class.